data/datasets/TSSB-3M/README.md

```diff
@@ -0,0 +1,14 @@
+# Dataset summary
+
+This dataset contains over 3 million isolated single statement bug fixes. Each
+bug fix is related to a commit in a public Python that does not change more than
+a single statement
+
+1. The original dataset comes from the
+   [TSSB-3M](https://zenodo.org/record/5845439) dataset
+2. By requesting the GitHub api to obtain the commit message, we expand and
+   create a new dataset
+   [TSSB-3M-ext](https://huggingface.co/datasets/zirui3/TSSB-3M-ext)
+3. Convert `TSSB-3M-ext` into instruction form to form the
+   [TSSB-3M-instruction](https://huggingface.co/datasets/zirui3/TSSB-3M-instructions)
```

Comment:
for some reason the HF dataset preview does not work (could be a caching problem that will just disappear on its own though)

Reply:
This is due to a problem with the. gz format. The data has been updated and can now be viewed normally